lesson07/assignment/vue-cli/src/views/Home.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class='Home'>
+    <app-jumbo></app-jumbo>
+    <app-caro :card='decks'></app-caro>
+
+
+
+  </div>
+</template>
+
+<script>
+
+//tried moving decks variable to main.js and importing into .vue
+//import {decks} from 'main.js'
+//get unused vars error - how do I fix this to only have this variable in one place?
```

Comment:
The shared data, `decks`, will need to be in a file that can be imported into each of these view files. You're on the right track with the syntax. main.js isn't the correct file, though. Usually, projects have a directory or file for shared data or functionality. I'll do an example of this and open a PR to your repo, so you can see how it would work.

Reply:
https://github.com/Headieh/html300/pull/10